src/main/scala/com/sksamuel/scapegoat/Feedback.scala

```diff
@@ -60,7 +60,7 @@ class Feedback(
     if (shouldPrint(warning)) {
       val snippetText = snippet.fold("")("\n  " + _ + "\n")
       val report = s"""|[scapegoat] $text
-                       |  $explanation$snippetText""".stripMargin
+                      |  $explanation$snippetText""".stripMargin
```

Comment:
Seems ok to me except for this, which doesn't look right. I'm wondering what happened there.

Reply:
Why would we care? Indeed it looks weird why would `scalafmt` perform a change like this.

But even with that change the output is the same, right?